portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,122 @@
+                            <a href="mailto:caroldotli@gmail.com"><i class="far fa-envelope fa-2x"></i></a>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="segment">
+            <div id="proj_segment">
+                <div class="segment_text">
+                    <h1>PROJECTS</h1>
+                    <div class="box">
+                        <p>
+                            On campus, I'm a Project Manager for <a href="https://hack4impact.org/">Hack4Impact</a>, a non-profit that creates socially impactful
+                            software for other non-profits, and I do full-stack development for <a href="https://thesign.al/">The Sign.al</a>, a group
+                            that destigmatizes failure through data-driven, technology-focused, and impactful projects. Last summer, I've conducted 
+                            <a href="https://penntoday.upenn.edu/news/smart-aviary-poised-break-new-ground-behavioral-research">research</a> for a computational
+                            neuroscience lab at Penn.
+                        </p>
+                    </div>
+
+                    <h2>AUTOMATIC ROOM ALLOCATOR</h2>
+                    <h3>for Ronald McDonald House Philadelphia &nbsp; • &nbsp;<a class="light_blue">Flask, PostgreSQL</a></h3>
+                    <!-- add later
+                    &nbsp; • &nbsp; <a><i class="fab fa-github"></a></i> -->
+                    <div class="box">
+                        <p>
+                            Room allocator for Ronald McDonald House client purposed to reduce manual entry and processing time of room requests. 
+                            Worked on UI/UX design, clientel relations, and full-stack development.
+                        </p>
+                    </div>
+
+                    <h2>PHILADELPHIA CONTRACT DATABASE</h2>
+                    <h3>for the Philadelphia City Controller &nbsp; • &nbsp;<a class="light_blue">Flask, PostgreSQL</a></h3>
+                    <div class="box">
+                        <p>
+                            Searchable database for the City of Philadelphia containing all city, lobbyist, and campaign finance contracts. Worked as 
+                            project manager, involving project structure breakdown and task allocation, full-stack development, clientel relations, 
+                            and UI/UX design.
+                        </p>
+                    </div>
+
+                    <h2>ANIMAL VOCALIZATION GENERATIVE NETWORK</h2>
+                    <h3>for Passerine Cowbirds &nbsp; • &nbsp;<a class="light_blue">Pandas, Tensorflow, Sci-Kit Learn</a></h3>
+                    <div class="box">
+                        <p>
+                            Generative neural network autoencoder using MDS and variational models purposed to automatically visualize male passerine songbird songbird
```

Comment:
I think you have 'songbird' twice in a row.

Also, this work is SO cool!! I just learned about autoencoders a few weeks ago :)

Reply:
Oops thanks for the catch :") hehe thanks